src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2018-2022  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const Dutch = {
+  language: "Dutch",
+  errors: {
+    deviceDisconnected: "Verbinding met toetsenbord verbroken",
+    saveFile: "Fout tijdens opslaan van bestand: {{ error }}",
+  },
+  components: {
+    layer: "Laag {{index}}",
+    layerRaw: "Laag",
+    none: "Geen",
+    type: "Type",
+    save: {
+      success: "Opgeslaan!",
+      saveChanges: "Aanpassingen Opslaan",
+    },
+    pickerColorButton: "Verander kleur",
+    loading: "Data lezen van toestel...",
+    logo: {
+      altText: "Chrysalis logo",
+    },
+  },
+  dialog: {
+    ok: "Ok",
+    cancel: "Annuleer",
+    close: "Sluiten",
+  },
+  app: {
+    menu: {
+      "focus-not-detected": "Welkom",
+      systemInfo: "Probleem melden",
+      editor: "Layout & Kleurmap Editor",
+      layoutEditor: "Layout Editor",
+      colormapEditor: "Kleurmap Editor",
+      macroEditor: "Macro Editor",
+      firmwareUpdate: "Firmware Update",
+      keyboardSettings: "Toetsenbord Instellingen",
+      preferences: "Voorkeuren",
+      selectAKeyboard: "Selecteer een toetsenbord",
+      layoutCard: "Layout Kaart",
```

Comment:
This new translation doesn't read nicely, but if we change “layout” to `indeling` this has to be consistent.  Perhaps a different ordering or word other than `kaart` can be used?

```suggestion
      layoutCard: "Indeling Kaart",
```